inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-golang/dataproxy/discoverer.go

```diff
@@ -180,8 +180,6 @@ func (d *dataProxyDiscoverer) update() {
 				d.lookup()
 			case <-stopCh:
 				return
-			default:
```

Comment:
If performance is affected by sleep, would it be better to simply use default: return?